crates/aptos-dkg/benches/crypto.rs

```diff
@@ -49,6 +50,14 @@ pub fn crypto_group(c: &mut Criterion) {
         accumulator_poly_slow(thresh, &mut group);
     }
 
+    const AVG_CASE: usize = 74;
```

Comment:
Should there be a comment for how this is derived?

Reply:
Addressed.